vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -803,30 +807,33 @@ void LedgerImpl::LoadPublishersListCallback(
   }
 }
 
-void LedgerImpl::RefreshPublishersList(bool retryAfterError) {
+void LedgerImpl::RefreshPublishersList(bool retryAfterError, bool immediately) {
   uint64_t start_timer_in{ 0ull };
 
   if (last_pub_load_timer_id_ != 0) {
     // timer in progress
     return;
   }
 
-  if (retryAfterError) {
-    start_timer_in = retryRequestSetup(300, 3600);
+  uint64_t lastLoadTimestamp =
+        bat_publishers_->getLastPublishersListLoadTimestamp();
+
+  if (lastLoadTimestamp == 0ull || immediately) {
+    start_timer_in = 0ull;
+  } else if (retryAfterError) {
+    start_timer_in = retryRequestSetup(60, 300);
```

Comment:
retry timer was reduced to 1-5min as it's really important to get this list asap. Endpoint is behind CDN so we should be good here